.editorconfig

```diff
@@ -0,0 +1,12 @@
+# top-most EditorConfig file
+root = true
+
+[*.cs]
+indent_style = tab
```

Comment:
Tabs rather that spaces? Didn't you see that episode of "Silicon Valley" :-) ? Well, OK, you will get it your way.

Reply:
I don’t care about tabs vs spaces. I configured it match the existing files.

Reply:
If you want to use space I suggest you make a separate PR and fix all existing files.